docs/autoencoders.md

```diff
@@ -0,0 +1,90 @@
+# Autoencoders
+
+An Autoencoder is a type of Artificial Neural Network, used to learn efficient data representations (encoding) of unlabeled data. 
+
+It mainly consist of 2 components: an encoder and a decoder. The encoder compresses the input into a lower dimensional code, the decoder then reconstructs the input only using this code.
+
+### Autoencoder Pipelines
+
+Numalogic provides two types of pipelines for Autoencoders.
```

Comment:
We can add that: 
these pipelines serves as a wrapper around the base network models, making it easier to train, predict and generate scores. Also, this module follows the sklearn API